webpack.config.js

```diff
@@ -3,16 +3,17 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 const config = {
+  mode: 'development',
```

Comment:
The mode should be determined by the deployment environment variables I think.

Reply:
It is being defined by scripts this is just for cases if the user runs a specific command. (which would only happen if they are in dev)